scalikejdbc-core/src/main/scala/scalikejdbc/DBConnectionAttributesWiredResultSet.scala

```diff
@@ -8,30 +8,37 @@ import java.util
 import java.util.Calendar
 
 /**
- * TimeZone conversion proxy of ResultSet
+ * ResultSet proxy which holds current DBConnectionAttributes.
  */
-case class ResultSetProxy(underlying: ResultSet, connectionAttributes: DBConnectionAttributes) extends ResultSet {
+private[scalikejdbc] class DBConnectionAttributesWiredResultSet(
```

Comment:
This class should be internal and accessor for underlying and connAttributes are unnecessary.